src/views/PeopleSearch/components/PeopleSearchResult/PeopleSearchResult.module.scss

```
@@ -1,5 +1,11 @@
 @import '../../../../vars';
 
+.resultContainer {
```

```suggestion
.result_container {
```
Also here (see below comment)